files/jquery.typing/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "jquery-typing",
+  "repo": "narfdotpl/jquery-typing",
+  "files": {
+    "basePath": "plugin/",
+    "include": ["*.min.js"]
```

Comment:
The filenames contain versions in them, we cant update those because then our version aliasing will break